alot/settings/manager.py

```diff
@@ -476,21 +476,21 @@ def get_accounts(self):
         """
         return self._accounts
 
-    def get_account_by_address(self, address, return_default=False):
+    def get_first_account_with_address(self, address, return_default=False):
```

Comment:
how about `account_matching_address` ?

Reply:
breaks with the get_* scheme of much of the rest of the class, but if that's fine with you I'll use that.